src/System.Reflection.TypeExtensions/src/System.Reflection.TypeExtensions.builds

```diff
@@ -10,6 +10,9 @@
     <Project Include="System.Reflection.TypeExtensions.csproj">
       <TargetGroup>netcore50aot</TargetGroup>
     </Project> -->
+    <Project Include="System.Reflection.TypeExtensions.csproj">
```

Comment:
When it comes to evaluate compatible configurations NuGet prefers the runtime specific one - in this case runtimes/aot/lib/netcore50 over lib/uap101.